src/Aspire.Hosting/Dashboard/DashboardServiceData.cs

```diff
@@ -77,6 +81,18 @@ public async ValueTask DisposeAsync()
         _cts.Dispose();
     }
 
+    internal async Task ExecuteCommandAsync(string resourceId, string type)
+    {
+        if (_resourcePublisher.TryGetResource(resourceId, out _, out var resource))
```

Comment:
If resource does not exist, or the command does not exist, perhaps we should log that as a warning/error.